fern/tool-calling/tool-calling.mdx

```diff
@@ -9,7 +9,6 @@ With Composio, your AI apps and agents gain access to over 250 integrations, inc
 
 ## Overview
 
-{/* TODO: make a flow for our given example */}
 ![Tool calling flow](../assets/images/tool-calling.png)
```

Comment:
Thank you for removing the TODO comment. While you're updating this section, I'd like to suggest two improvements:

1. **Enhance the image alt text for better accessibility:**
```markdown
![Diagram showing the tool-calling flow process in Composio](../assets/images/tool-calling.png)
```

2. **Add context between the image and code example to improve documentation flow:**
```markdown
![Diagram showing the tool-calling flow process in Composio](../assets/images/tool-calling.png)

The diagram above illustrates how Composio connects your AI application to external tools through our tool-calling API. Here's the above example in code:
```

These changes would improve accessibility and provide better context for users reading the documentation.